docker-compose.yml

```diff
@@ -1,46 +1,36 @@
 version: "3.5"
 services:
   node:
-    image: "node.hopr.public.dappnode.eth:1.0.27"
+    image: "node.hopr.public.dappnode.eth:1.0.19"
     build:
       context: .
       args:
-        UPSTREAM_VERSION: "2.0.7@sha256:cb63467df80840dc20b5f9f6d55977ede6e2902014dc67c86f3bbed3b69815ef"
+        UPSTREAM_VERSION: latest
     ports:
       - "9091:9091/tcp"
       - "9091:9091/udp"
       - "3001"
     volumes:
-      - "db:/app/hoprd-db"
+      - "data:/app/data"
```

Comment:
Changing the volume name from `db` to `data` will provoke the users to loose all their data, including the identity file which is stored within the volume.
With previous versions, the docker image was able to attach a shell, and enter into the `db` volume while the docker-compose was running. With the rust version, the user won't be able to do it.
It will need to stop the app, and create a manual docker-compose which has mounted as well the volume. 
I would propose not to use docker volumes but instead use a path within the dappNode. That would help administration tasks like using a different identity_file or cleaning database.

Reply:
- regarding DB data that is expected. Because we're using a different DB in 2.1
- regarding the identity, we might need to ask users to backup the identity file and restore the ID file after the update

Reply:
@jaguaras keep this breaking change in mind